RegisterAPI/DataAccess/CassandraHandler.cs

```diff
@@ -0,0 +1,72 @@
+﻿/// <summary>
+/// Developer: ShyamSk
+/// </summary>
+
+namespace RegisterAPI.DataAccess
+{
+    using Cassandra;
+    using Cassandra.Mapping;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+    using RegisterAPI.Abstract;
+    using RegisterAPI.Models;
+    using System;
+
+    /// <summary>
+    /// Cassandra DB Handler
+    /// </summary>
+    public class CassandraHandler : IDataAccess
+    {
+        private readonly ILogger<CassandraHandler> _logger;
+        private IConfiguration _configuration;
+
+        public CassandraHandler(ILogger<CassandraHandler> logger, IConfiguration configuration)
+        {
+            _logger = logger;
+            _configuration = configuration;
+        }
+        /// <summary>
+        /// Saves User Profile
+        /// </summary>
+        /// <param name="profile"></param>
+        /// <returns></returns>
+        public Guid SaveProfile(UserProfile profile)
+        {
+            _logger.LogInformation("Cassandra - Inserting profile ....");
+            UserProfile profileD = new UserProfile();
+
+            try
+            {
+                IConfiguration cassandraConfig = _configuration.GetSection("Cassandra");
+                string contactPoints = cassandraConfig.GetValue<string>("host");// "127.0.0.1,127.0.0.2,127.0.0.3";

```

Comment:
configuration and data connection should not be part of method, will share you the framework